src/components/ColorStop/index.css

```diff
@@ -14,6 +14,10 @@
     top: 8px;
 }
 
+.cs div.with-opacity {
```

Comment:
Is this meant to be controlled by the consumer? e.g override this class display?

WDYT on allow this via a new property? `showStopColorOpacity`

Reply:
What do you think of an optional new theme for the preview and colorStops?
![image](https://user-images.githubusercontent.com/79632638/140709569-280577c4-4610-43b6-ab03-c4be56ab2913.png)

Reply:
Sorry for the late response, I was kinda into the fact that this lib would come with minimal CSS since the fact it would fit your styling is kinda low.